### ISSUE TITLE
Ensures buttons have discernible text.

### ISSUE THREAD

**Issue by msftedad** [ant-design/ant-design#45057]
### Reproduction link

[https://codepen.io/afc163-1472555193/pen/oNXqWGP](https://codepen.io/afc163-1472555193/pen/oNXqWGP)

### Steps to reproduce

Hit the URL: DatePicker - Ant Design Demo (codepen.io).
TAB to any calendar filed and expand it.
Run accessibility insights for web.
Verify whether buttons have discernible text or not.

### What is expected?

Buttons should have discernible text.

### What is actually happening?

Buttons do not have a discernible text.

| Environment | Info |
| --- | --- |
| antd | 4.24.13 |
| React | 17.0.2 |
| System | Windows 11 Enterprise 22H2 Version: 22621.2283 |
| Browser | Edge Version 116.0.1938.81 (Official build) (64-bit) |

Attachments:
![Bug_212636_Ensures buttons have discernible text](https://github.com/ant-design/ant-design/assets/93735775/2f9c94a3-d288-4444-ab72-4b04077f5300)

**Comment by MadCcc** [comment]
Any advice? These buttons do not have a text content because they contained only icons.

**Comment by msftedad** [comment]
If no text is provided the users may not know or get confused what that symbols are. Thanks

**Comment by afc163** [comment]
We could add `aria-label` on it.

**Comment by jjy05** [comment]
> Any advice? These buttons do not have a text content because they contained only icons.

Display some descriptions when hover? For example use `title`

**Comment by afc163** [comment]
fixed in https://github.com/ant-design/ant-design/pull/49333
